packages/components/src/Tooltip/src/TooltipTrigger.tsx

```diff
@@ -0,0 +1,55 @@
+import { TooltipTrigger as RACTooltipTrigger, type TooltipProps as RACTooltipProps, type TooltipTriggerComponentProps as RACTooltipTriggerProps } from "react-aria-components";
+
+import { TooltipTriggerContext } from "./TooltipTriggerContext.ts";
+
+export type TooltipPlacement = "start" | "end" | "right" | "left" | "top" | "bottom";
+
+type PropsToPick = "shouldFlip" | "containerPadding" | "offset" | "crossOffset" | "isOpen" | "defaultOpen" | "onOpenChange";
+
+export interface TooltipTriggerProps extends RACTooltipTriggerProps, Pick<RACTooltipProps, PropsToPick> {
+    /**
+   * The placement of the element with respect to its anchor element.
+   *
+   * @default 'top'
+   */
+    placement?: TooltipPlacement;
+}
+
+/**
+ * A TooltipTrigger wraps a trigger element and Tooltip, handling visibility and positioning.
+ *
+ * [View Documentation](TODO)
+ */
+export function TooltipTrigger(props: TooltipTriggerProps) {
```

Comment:
[Question] We don't need to expose the ref for this one?

Reply:
nope, no ref since the ref is the trigger that the user adds, plus they can put a ref on the Button for example.

Reply:
TooltipTrigger doesn't render any html element, so there is nothing to add a ref to. If you wanted to add a ref to the button, you need to do it
```
<TooltipTrigger>
    <Button ref={ref} >Click me</Button>
    <Tooltip>TooltipContent</Tooltip>
</TooltipTrigger>
```